libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -921,7 +921,7 @@ void AP_CRSF_Telem::calc_flight_mode()
             sizeof(AP_CRSF_Telem::FlightModeFrame), 
             "%s%s", 
             notify->get_flight_mode_str(), 
-            hal.util->get_soft_armed() ? "" : "*"
+            !rc().use_crsf_lq_as_rssi() || hal.util->get_soft_armed() ? "" : "*"
```

Comment:
I dont think this is the correct option and logic

Reply:
oops, copy and forgot, I just corrected. 
I think the logic is right, I just flipped it so it's now easier to read.